includes/admin/class-wcmypa-admin.php

```diff
@@ -989,14 +990,22 @@ private function getConfirmationData(DeliveryOptions $deliveryOptions): ?array
             ];
         }
 
-        return [
-            __("Delivery type:", "woocommerce-myparcel") => WCMP_Data::getDeliveryTypesHuman()[$deliveryOptions->getDeliveryType()],
-            __("Date:", 'woocommerce')                   => wc_format_datetime(new WC_DateTime($deliveryOptions->getDate())),
-            __("Extra options:", "woocommerce-myparcel") =>
-                sprintf("%s<br>%s",
-                        $deliveryOptions->getShipmentOptions()->hasSignature() ? $signatureTitle : null,
-                        $deliveryOptions->getShipmentOptions()->hasOnlyRecipient() ? $onlyRecipientTitle : null)
+        $confirmationData = [
+            __("delivery_type:", "woocommerce-myparcel") => WCMP_Data::getDeliveryTypesHuman()[$deliveryOptions->getDeliveryType()],
         ];
+
+        if (WCMYPA()->setting_collection->isEnabled(WCMYPA_Settings::SETTING_SHOW_DELIVERY_DAY)) {
+            $confirmationData[__("Date:", 'woocommerce')] = wc_format_datetime(new WC_DateTime($deliveryOptions->getDate()));;
+        }
+
+        if ($hasSignature || $hasOnlyRecipient) {
+            $confirmationData[__("extra_options:", "woocommerce-myparcel")] =
```

Comment:
Ik zou misschien de `:` weg laten, achter `extra_options`. Het probleem nu is namelijk als iemand vind dat hij weg moet dan moet er alsnog een aanpassing gedaan worden in de code.

Reply:
Die moet zeeeeker weg

Reply:
aha oke